### PR TITLE
Update default config file with `env` for everything and added docs

### PR DESCRIPTION
**Pull request by stayallive**

### CODE REVIEW

File: config/sentry.php [hunk 1]
```diff
@@ -9,80 +14,89 @@
...
     'environment' => env('SENTRY_ENVIRONMENT'),
 
+    // @see: https://docs.sentry.io/platforms/php/guides/laravel/configuration/options/#sample-rate
+    'sample_rate' => env('SENTRY_SAMPLE_RATE') === null ? 1.0 : (float)env('SENTRY_SAMPLE_RATE'),
+
+    // @see: https://docs.sentry.io/platforms/php/guides/laravel/configuration/options/#traces-sample-rate
+    'traces_sample_rate' => env('SENTRY_TRACES_SAMPLE_RATE') === null ? null : (float)env('SENTRY_TRACES_SAMPLE_RATE'),
+
+    // @see: https://docs.sentry.io/platforms/php/guides/laravel/configuration/options/#traces-sample-rate
+    'traces_sample_rate' => env('SENTRY_TRACES_SAMPLE_RATE') === null ? null : (float)env('SENTRY_TRACES_SAMPLE_RATE'),
```
Comment (stayallive) [inline config/sentry.php:27]: One should be plenty!

```suggestion
    // @see: https://docs.sentry.io/platforms/php/guides/laravel/configuration/options/#traces-sample-rate
    'traces_sample_rate' => env('SENTRY_TRACES_SAMPLE_RATE') === null ? null : (float)env('SENTRY_TRACES_SAMPLE_RATE'),
```

Comment (cleptric) [reply config/sentry.php:27]: Ups!

**Review by cleptric — APPROVED** [review]
Merged master and added the new storage config as well.

**Comment by stayallive** [comment]
I like it! 💪

**Pull request merged**